doc/source/user/tutorial-svd.rst

```diff
@@ -0,0 +1,440 @@
+================================================
+Tutorial: Linear algebra on n-dimensional arrays
+================================================
+
+.. currentmodule:: numpy
+
+.. testsetup::
+
+   import numpy as np
+   np.random.seed(1)
+
+**Prerequisites**
+
+Before reading this tutorial, you should know a bit of Python. If you
+would like to refresh your memory, take a look at the `Python
+tutorial <https://docs.python.org/tutorial/>`__.
+
+If you want to be able to run the examples in this tutorial, you should also have
+`matplotlib <https://matplotlib.org/>`_ and `SciPy <https://scipy.org>`_ installed on your computer.
+
+**Learner profile**
+
+This tutorial is for people who have a basic understanding of linear
+algebra and arrays in NumPy and want to understand how n-dimensional
+(:math:`n>=2`) arrays are represented and can be manipulated. In particular, if
+you don't know how to apply common functions to n-dimensional arrays (without
+using for-loops), or if you want to understand axis and shape properties for
+n-dimensional arrays, this tutorial might be of help.
+
+**Learning Objectives**
+
+After this tutorial, you should be able to:
+
+- Understand the difference between one-, two- and n-dimensional arrays in NumPy;
+- Understand how to apply some linear algebra operations to n-dimensional arrays without using for-loops;
+- Understand axis and shape properties for n-dimensional arrays.
+
+**Content**
+
+In this tutorial, we will use a `matrix decomposition <https://en.wikipedia.org/wiki/Matrix_decomposition>`_ from linear algebra, the Singular Value Decomposition,  to
+generate a compressed approximation of an image. We'll use the ``face`` image from the
+`scipy.misc` module:
+
+    >>> from scipy import misc
+    >>> img = misc.face()
+
+.. note::
+
+   If you prefer, you can use your own image as you work through this tutorial. In
+   order to transform your image into a NumPy array that can be manipulated, you can
+   use the ``imread`` function from the `matplotlib.pyplot` submodule. Alternatively,
+   you can use the ``imread`` function from the `imageio library
+   <https://imageio.readthedocs.io/en/stable/userapi.html#imageio.imread>`_. Be aware
```

Comment:
It would be neat to use `intersphinx` here

Reply:
I haven't done this yet, but I'll try looking into it, thanks.

Reply:
I think this would mean (approximately, I didn't test it):
- adding https://imageio.readthedocs.io/en/stable to the intersphinx section of the `conf.py`
- changing the link to
  ```
  `imread <imageio.imread>`
  or
  `imageio.imread`
  ```

Reply:
Or `~imageio.imread` I think too?

Reply:
That worked, thanks!